File: URI_Online_Judge_1080.cpp
#include <iostream>
using namespace std;
 
int main() {
 
    int N,highest=0,position;
    
    for(int i=0; i<100; i++){
    	cin>>N;
    	if(N>highest){
    		highest = N;
    		position = i + 1;
		}
    	
	}
	
	cout<<highest<<endl<<position<<endl;
	
    return 0;
}

File: URI_Online_Judge_1080.java
import java.util.Scanner;

public class Main {

	public static void main(String[] args) {

		int N, highest = 0, position = 0;

		Scanner in = new Scanner(System.in);

		for (int i = 0; i < 100; i++) {
			N = in.nextInt();
			if (N > highest) {
				highest = N;
				position = i + 1;
			}

		}

		System.out.println(highest + "\n" + position);

	}
}